cmd/oras/internal/option/parser.go

```diff
@@ -17,18 +17,20 @@ package option
 
 import (
 	"reflect"
+
+	"github.com/spf13/cobra"
 )
 
 // FlagParser parses flags in an option.
 type FlagParser interface {
-	Parse() error
+	Parse(cmd *cobra.Command) error
 }
 
 // Parse parses applicable fields of the passed-in option pointer and returns
 // error during parsing.
-func Parse(optsPtr interface{}) error {
+func Parse(optsPtr interface{}, cmd *cobra.Command) error {
```

Comment:
Can we put `cmd` as the first argument for consistency?
```suggestion
func Parse(cmd *cobra.Command, optsPtr any) error {
```